binary_search/binarysearch.cpp

```diff
@@ -0,0 +1,42 @@
+#include<iostream>
+
+using namespace std;
+
+int binary_search_iterative(int array[], int start, int end, int find)
+{
+	while(end >= start)
```

Comment:
Please make this, `while (...) {` as per standard cpp guidelines. Same changes in rest of the file.